src/network/live_configuration/live_configuration.rs

```diff
@@ -84,58 +79,72 @@ impl LiveConfigurationImpl {
             offline_mode,
         }
     }
-}
 
-impl LiveConfiguration for LiveConfigurationImpl {
     fn get_configuration(&self) -> Result<Configuration> {
+        // TODO: Can we return a reference instead?
```

Comment:
I'm just moving the comment we already have in several lines (_"// TODO: we do not want to clone here"_) to the top of the function. It's not the purpose of this PR to implement (or not) this.